### DISCUSSION TITLE
Where can i find the piper configuration this repo is using?

### DISCUSSION THREAD

**Discussion by KAGEYAM4** [ken107/read-aloud#396] [Q&A] [answered]
I am using speech-dispatcher (https://github.com/brailcom/speechd) on linux. But the configuration of piper i am using takes 2-3 seconds pause between sentences. I was able to improve this by addidig `GenericDelimiters "˨"` which i found here - https://github.com/ken107/read-aloud/issues/375#issuecomment-1937517761

As the above link states, i am also having big pause between sentences.

So does read-alound uses, `spd-say`? What config is it using for this? I lookd at source code but couldn't found anything useful.

**Comment by ken107** [comment, marked as answer]
Hi, the piper related code is in this repository
https://github.com/ken107/piper-browser-extension in the _web_ folder.

The code performs inferencing directly inside the browser using onnxruntime-web, to synthesize speech. It doesnt use spd-say or any third-party tool.

I'm guessing the 2-3 second pause you're experiencing is the time it takes to synthesize the next sentence. Our implementation deals with that by pre-synthesizing the next sentence while the current sentence is being spoken. Your tool will need to support this 'prefetching' strategy.

**Reply by KAGEYAM4** [reply]
I will look into 'prefetching'. Thank you.
